### ISSUE TITLE
Question about the data preprocess in dataloader.py

### ISSUE THREAD

**Issue by Serenade-ajp** [wvangansbeke/Sparse-Depth-Completion#5]
Hello!
Thank you so much to share your code publicly for free.
I am confused about the following code in dataloader.py, probably from the line 174 to 180:

input, gt = self.totensor(sparse_depth_np).float(), self.totensor(gt_np).float()

        if self.normal:
            # Put in {0-1} range and then normalize
            input = input/self.max_depth
            gt = gt/self.max_depth
            # input = self.depth_norm(input)
we know that totensor in pytorch can converts a PIL Image in range [0,255] to a torch.FloatTensor in the range of [0.0, 1.0] automatically, but you also do the normalization by divided by the max_depth, I wonder whether this transformation is is proper?
Wish your reply, thank you!

**Comment by wvangansbeke** [comment]
Hi @Serenade-ajp,

I get the confusion. The `sparse depth` matrix does not get normalized since it is a numpy matrix. For a PIL image this is indeed the case. It could have been changed since the latest release of torchvision but I don't think so. Yes I'm right. Check the last line in the documentation underneath.

![totensor_doc](https://user-images.githubusercontent.com/9694230/59829215-1ec49a80-933d-11e9-9183-c711e83e410c.png)

Kind regards,
Wouter

**Comment by Serenade-ajp** [comment]
@wvangansbeke 
Yes, I have checked it, you are right!
Thanks for your reply.
